src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_params.py

```diff
@@ -597,6 +597,20 @@ def _configure_security_policy_storage_params(arg_ctx):
         c.argument('dtu', help='TThe total shared DTU for the elastic eool.')
         c.argument('storage_mb', help='Storage limit for the elastic pool in MB.')
 
+    #####
+    #           sql elastic-pool op
+    #####
+    with self.argument_context('sql elastic-pool op') as c:
+        c.argument('elastic_pool_name',
+                   options_list=['--elastic-pool'],
+                   required=True,
+                   help='Name of the Azure SQL Elastic Pool.')
+
+        c.argument('operation_id',
+                   options_list=['--name', '-n'],
+                   required=True,
```

Comment:
This should not be needed. The CLI will detect that the parameter is required.

Reply:
@tjprescott I will remove this "required" property. same for the above one.